Assets/Scripts/Player/PlayerParticleEffects.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
```

Comment:
I think this should be moved elsewhere. If this class is used to handle all particle effects for the player it's going to be tightly coupled with a bunch of unrelated things and that's generally best to avoid because it becomes difficult to maintain/refactor. I'd suggest you move the particle effect logic to the classes that use them (probably most of the spell classes).